recipe/meta.yaml

```diff
@@ -165,118 +169,70 @@ outputs:
-        # constraint is boto3~=1.10<1.11 and no watchtower
-        - boto3 >=1.12.0,<2.0.0
-        - watchtower 0.7.3
-    test:
-      imports:
-        - airflow
-        - boto3
-        - watchtower
-        - airflow.contrib.hooks.aws_hook
-
-  - name: {{ name }}-with-azure_blob_storage
-    requirements:
-      host:
-        - python
-      run:
-        - python
-        - {{ pin_subpackage(name, max_pin='x.x.x.x.x.x') }}
-        - azure-storage >=0.34.0,<0.37.0
-        - azure-storage-common
-
-    test:
-      imports:
-        - airflow
-        - azure
-        - azure.storage
-        - azure.storage.blob
-        - airflow.contrib.hooks.wasb_hook
-
-  - name: {{ name }}-with-azure_data_lake
-    requirements:
-      host:
-        - python
-      run:
-        - python
-        - {{ pin_subpackage(name, max_pin='x.x.x.x.x.x') }}
-        - azure-datalake-store >=0.0.45
-        - azure-mgmt-datalake-store >=0.5.0
-        - azure-mgmt-resource >=2.2.0
-    test:
-      imports:
-        - airflow
-        - azure.datalake.store
-        - airflow.contrib.hooks.azure_data_lake_hook
+        - airflow.lineage
+        - airflow.lineage.entities
+      commands:
+        - pip check
+      requires:
+        - pip
 
-  - name: {{ name }}-with-azure_cosmos
+  - name: {{ name }}-with-apache-beam
     requirements:
       host:
         - python
       run:
         - python
         - {{ pin_subpackage(name, max_pin='x.x.x.x.x.x') }}
-        - azure-cosmos >=3.0.1,<4
+        - apache-beam
     test:
       imports:
         - airflow
-        - azure.cosmos
-        - airflow.contrib.hooks.azure_cosmos_hook
-
-  # Note: no azure-mgmt-containerinstance feedstock
-  #- name: {{ name }}-with-azure-mgmt-containerinstance
-  #  requirements:
-  #    host:
-  #      - python
-  #    run:
-  #      - python
-  #      - {{ pin_subpackage(name, max_pin='x.x.x.x.x.x') }}
-  #      - azure-mgmt-containerinstance >=1.5.0,<2
-  #  test:
-  #    imports:
-  #      - airflow
-  #      - azure.mgmt.containerinstance
+        - apache_beam
+      commands:
+        - pip check
+      requires:
+        - pip
 
-  - name: {{ name }}-with-cassandra
-    requirements:
-      host:
-        - python
-      run:
-        - python
-        - {{ pin_subpackage(name, max_pin='x.x.x.x.x.x') }}
-        - cassandra-driver >=3.13.0,<3.21.0
-    test:
-      imports:
-        - airflow
-        - cassandra
-        - airflow.contrib.hooks.cassandra_hook
+#  apache-airflow-providers-apache-hdfs is not yet available
+#  - name: {{ name }}-with-apache-webhdfs
+#    requirements:
+#      host:
+#        - python
+#      run:
+#        - python
+#        - {{ pin_subpackage(name, max_pin='x.x.x.x.x.x') }}
+#        - python-hdfs >=2.0.4
+#        - apache-airflow-providers-apache-hdfs
+#    test:
+#      imports:
+#        - airflow
+#        - hdfs
+#        - airflow.hooks.webhdfs_hook
+#      commands:
+#        - pip check
+#      requires:
+#        - pip
```

Comment:
This requires `apache-airflow-providers-apache-hdfs`, which in turn requires `apache-airflow`, so I'll uncomment this in a later build.